### DISCUSSION TITLE
Is there anyone who can help me in building it up and scaling it up? What fields you can help me??

### DISCUSSION THREAD

**Discussion by LavkushSolanki** [LavkushSolanki/Trad#4] [Q&A] [answered]
Is there anyone who can help me in building it up and scaling it up? What fields you can help me??

**Comment by RahulKumarSingh123** [comment, marked as answer]
Hii buddy I can aid you in scaling ,I have knowledge about Aws

**Comment by shige** [comment]
## 相談事項: セキュリティ Issue の担当者について

#4939 [Security] Authorization Bypass - Cross-Workspace Secret Access の対応担当者を決めたいです。

現在 @toyamarinyon さんがアサインされていますが、このまま外山さんに対応いただくか、@WashizuRyo さんに対応をお願いするか、ご相談させてください。

深刻度が「緊急」となっているため、早めに担当を確定して対応を進めたいと考えています。
